### ISSUE TITLE
Add option to pay song instantly on button press

### ISSUE THREAD

**Issue by Shane8johnson** [whitesoup12/remote-falcon-tracker#20]
I would love a mode that does not use a queue and plays the requested song instantly. Is there any way to do this currently?
*EDIT: While reading through the wiki, it seems the request I am wanting is an option to Allow viewer requests to be interrupted.

**Comment by whitesoup12** [comment]
@Shane8johnson, that option would only interrupt the scheduled playlist. A viewer request will never be interrupted. RF did have an “On Demand” mode at one time that immediately played any request that came in, but it wasn’t ideal and caused more “chaos” than anything. I’ll still tag this as an enhancement though and we’ll see where it goes. 🙂

**Comment by Shane8johnson** [comment]
@whitesoup12 I think it would be good to have the option listed, even if there are next to none that use it (although I would for sure). The reason I request this is because I started back in LOR and built a physical jukebox with buttons and two [Input Pups](http://store.lightorama.com/inputpup.html). People loved it! If they got tired of a song, they just go pick a different one. I then moved over to Xlights this year and am using the guestbuttons.html file that Keith so graciously added as per my request and the ability to bypass the password for guest pages. This is all self hosted so I was looking at your service since it does not require any port forwarding or static IPs.
